deployment/helm/skaha/sample-local-values.yaml

```diff
@@ -2,14 +2,19 @@
 
 # Skaha web service deployment
 deployment:
-  hostname: example.org # change this
+  hostname: example.org # Change this
   skaha:
+    image: images.opencadc.org/platform/skaha:0.16.1
     imagePullPolicy: IfNotPresent
     # Space delimited list of allowed Image Registry hosts.  These hosts should match the hosts in the User Session images.
     registryHosts: "images.canfar.net"
+    maxUserSessions: "5"
```

Comment:
Please remove this as it's documented in the main `values.yaml` file.

Reply:
Removed